src/components/button/_button.scss

```diff
@@ -305,7 +305,7 @@
 
   .#{$prefix}--btn--sm {
     @include letter-spacing;
-    height: auto;
+    height: rem(32px);
```

Comment:
This is a bit different than the current theme height application:
https://github.com/IBM/carbon-components/blob/98bae993a38f43dc45e6a23b5302f882926a0ab1/src/components/button/_button.scss#L163

Reply:
the output is identical (2rem) as our base font size is 16px, unless I am mistaken. Please correct me if that's not the case

Reply:
Yup, it's the same. Just pointing it out for consistency 👍